docs/reference/cluster/nodes-reload-secure-settings.asciidoc

```diff
@@ -1,9 +1,9 @@
 [[cluster-nodes-reload-secure-settings]]
 == Nodes Reload Secure Settings
 
-The cluster nodes reload secure settings API is used to re-read the
-local node's encrypted keystore. Specifically, it will prompt the keystore
-decryption and reading across the cluster. The keystore's plain content is
+The cluster nodes reload secure settings API is used to re-read the local node's encrypted keystore.
+Specifically, it will prompt the keystore decryption and reading across the cluster.
+The keystore's decrypted content is
 used to reinitialize all compatible plugins. A compatible plugin can be
```

Comment:
I think this existing wording is deceptive. Plugin's can't be reinitialized. Rather, the consumers of the keystore settings are notified of the updated value.

Reply:
I went ahead and removed the existing additional information. It felt more like describing internal implementation than adding something of value for the users/API consumers.